### PR TITLE
Feature: Validate url when click into url

### PR DESCRIPTION
**Pull request by TranHungKT**

### CODE REVIEW

File: router/route.go [hunk 1]
```diff
@@ -22,10 +23,13 @@ func InitGin() {
 
 func HTMLRender(router *gin.Engine) {
 	router.Static("/static", "./static")
-	router.LoadHTMLGlob("./static/templates/*.html")
 	router.LoadHTMLGlob("./static/public/*.html")
-	router.GET("/subscriber/confirm-optin/:email", func(ctx *gin.Context) {
-		email := ctx.Param("email")
+	router.GET("/subscriber/confirm-optin/:nonceKey/:cipherEmailKey/:startedTime", func(ctx *gin.Context) {
+		nonce := ctx.Param("nonceKey")
```
Comment (TranHungKT) [inline router/route.go]: Let's make these string as const

File: static/public/confirmOptin.html [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+            </p>
         </form>
-        </div>
     </div>
+    
+</div>
 </body>
-</html>
+
+</html>
```
Comment (TranHungKT) [inline static/public/confirmOptin.html]: Please fix this

File: static/public/header.html [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+    integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
 
   <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
   <link rel="shortcut icon" href="http://localhost:3000/static/images/favicon.png" type="image/x-icon" />
 </head>
+
 <body class="body">
-{{end}}
+
+  {{end}}
```
Comment (TranHungKT) [inline static/public/header.html]: and this

File: static/public/outOfDateURL.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+<div class="template-container">
+    <h2>Oops...</h2>
+    <p>Your request is out of date</p>
+    <p>Please contact admin to get new url</p>
+</div>
+
+</body>
+
+</html>
```
Comment (TranHungKT) [inline static/public/outOfDateURL.html]: Same

File: static/public/successSubscription.html [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
-    </div>
+<div class="template-container">
+    <h2>Confirmed</h2>
+    <p>Subscriber successfully</p>
+    <p>Welcome to listmonk</p>
+</div>
 
 </body>
+
 </html>
```
Comment (TranHungKT) [inline static/public/successSubscription.html]: .

**Review by ngocanhtranhi — APPROVED** [review]

**Pull request merged**